libraries/core-react/src/components/Card/CardMedia.tsx

```diff
@@ -37,6 +34,7 @@ const StyledCardMedia = styled.div<CardMediaProps>`
           }
         `
       : css`
+          padding: 0 16px;
```

Comment:
Suggest updating this to use the token `spacings` values so its updates correctly if spacings change (the same way you've done in the other components).

Reply:
Good point, done.